src/integration.test.ts

```diff
@@ -3,18 +3,43 @@ import { genKeyPairAndSeed, SkynetClient } from "./index";
 const { publicKey, privateKey } = genKeyPairAndSeed();
 const client = new SkynetClient("https://siasky.dev");
 
-const datakey = "HelloWorld";
-const json = { data: "thisistext" };
+const dataKey = "HelloWorld";
+const maxint = BigInt("18446744073709551615"); // max uint64
 
 // skip - used to verify end-to-end flow
 describe.skip("siasky.dev end to end integration test", () => {
-  it("should be able to both setJSON and getJSON", async () => {
-    // set the file in the SkyDB
-    await client.db.setJSON(privateKey, datakey, json);
+  // NOTE: These tests must run sequentially.
```

Comment:
Why though? The tests are not dependant on each other right? 

If you move `const { publicKey, privateKey } = genKeyPairAndSeed();` inside of the test itself wouldn't you be able to run them in parallel? I don't think it's really an issue but I also don't see why we need to overcomplicate it and reference how you can run tests sequentially if they don't need to be run that way necessarily.

Reply:
Sure